dist/about.html

```diff
@@ -44,10 +44,17 @@
 					<div class="nav-collapse collapse">
 						<ul class="nav">
 							<li class=""><a href="index.html#services">Ratings</a></li>
+<<<<<<< HEAD
```

Comment:
It's getting worse xD

Reply:
Nope thats a problem on your side, you need to recompile as you merged stuff rather than pulling the fork update